util/internal/integrationtest/common/src/main/java/org/hibernate/search/util/impl/integrationtest/common/rule/IndexWorkCall.java

```diff
@@ -16,16 +16,16 @@
 
 class IndexWorkCall extends Call<IndexWorkCall> {
 
-	enum Operation {
+	enum WorkPhase {
 		PREPARE,
 		EXECUTE
 	}
 
 	private final String indexName;
-	private final Operation operation;
+	private final WorkPhase operation;
```

Comment:
Well that's nice, but maybe you could also remove the fields and variables accordingly :) And similarly change the error messages (see below).

Reply:
+1